### PR TITLE
Dockerfile: preinstall msmtp

### LINKED ISSUE #9: preinstall msmtp
In order to send reports, opendmarc needs a sendmail implementation. Currently there is none.

I'd recommend preinstalling msmtp to the docker images, as this can do TLS and SMTP-AUTH.

https://wiki.debian.org/msmtp
https://manpages.debian.org/bookworm/msmtp/msmtp.1.en.html

### PR DESCRIPTION
**Pull request by schildbach**
This implements `/usr/sbin/sendmail` which is needed by `FailureReports true`.

Also adds a new section about sending reports to the README.md.

Note that "aggregate reports" are outside the scope of my testing and this PR. `FailureReports true` generates reports immediately for individual emails.

I sadly didn't find a way around specifying the from address a second time in `/etc/msmtprc`, even though it's already known via `FailureReportsSentBy` OpenDMARC configuration option.

Resolves #9

### CODE REVIEW

**Comment by schildbach** [comment]
Ah, and I should add I've based most of my testing on the Debian image. However, since msmtp versions only differ minimally (Alpine: `1.8.25` vs Debian: `1.8.23`) I expect the same outcome.

File: alpine/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ RUN apk update \
  && apk upgrade \
  && apk add --no-cache \
         ca-certificates \
+        msmtp \
```
Comment (tyranron) [inline alpine/Dockerfile]: @schildbach these `Dockerfile`s are generated from the `Dockerfile.tmpl.php`, the header comment indicates this. So, these changes should be made in `Dockerfile.tmpl.php` and then populated via `make dockerfile` command.

File: README.md [hunk 2]
```diff
@@ -77,7 +77,26 @@ To see default OpenDMARC configuration of this Docker image just run:
 docker run --rm instrumentisto/opendmarc cat /etc/opendmarc/opendmarc.conf
 ```
 
+#### Sending reports
 
+The Docker images come with msmtp MTA preinstalled which you can use to send reports when requested via the `ruf` tag inside a DMARC record.
```
Comment (tyranron) [inline README.md]: @schildbach I wonder whether some tests could be added for this?

At least, that `sendmail` is present in the image and is working.

Ideally, that `FailureReports true` works as expected.

**Review by tyranron — CHANGES_REQUESTED** [review]

File: alpine/Dockerfile (hunk 1 above)
Comment (schildbach) [reply alpine/Dockerfile]: Sorry, wrapping in make is unexpected to me. I usually contain my builds in Docker.

Anyway, pushed a fix for this.

File: README.md (hunk 2 above)
Comment (schildbach) [reply README.md]: Also added a test for the presence of `/usr/sbin/sendmail`. I'm afraid a more thorough test is beyond my capabilities – I'm mainly a Java dev. I assume SMTP and DNS servers would need to be mocked, and a test case constructed that triggers such mail. Not a trivial task I'd say.

**Review by tyranron — APPROVED** [review]

**Comment by tyranron** [comment]
@schildbach released as [1.4.2-r36](https://github.com/instrumentisto/opendmarc-docker-image/releases/tag/1.4.2-r36).

**Pull request merged**